code/src/main/webapp/JavaScript/script.js

```diff
@@ -179,7 +179,8 @@ function attachSignin(element, negativeUser) {
     localStorage.setItem('idToken', idToken.toString());
     const params = new URLSearchParams()
     params.append('idToken', idToken);
-    fetch(new Request('/authentication-test', {method: 'POST', body: params}))
+
+    fetch(makeRequest(negativeUser, params))
     .then(response => response.text())
     .then(url => { if (url.length < 20) {
```

Comment:
Where does this magic value come from?

Reply:
looking back that may not have been the best solution

I will think of a better approach and put it in a separate PR

Reply:
Yep.